### DISCUSSION TITLE
CLI Uploading getting "Request failed with status code 404"

### DISCUSSION THREAD

**Discussion by dcglabs** [immich-app/immich#5108] [Q&A]
This issue is opened up over here: https://github.com/immich-app/CLI/issues/117
It seems something changed in v1.85.0 that is cause 404 errors on uploading to server. From my quick assessment it happens if item is already in the database. I use the CLI to upload photos more than once but use --album to have them tagged into multiple albums. 
Is there a better way to do this? Also, I can help further debugging this issue if someone can provide guidance on what to do.

Thanks

**Comment by holoskii** [comment]
Encountered the same issue. Attaching the log files I could get:
[docker_out.txt](https://github.com/immich-app/immich/files/13403051/docker_out.txt)
[cli_out.txt](https://github.com/immich-app/immich/files/13403052/cli_out.txt)

**Comment by narenarora** [comment]
Seeing a different issue with the newer version now

```
latest: Pulling from immich-app/immich-cli
8663204ce13b: Pull complete
d19b13245430: Pull complete
e4a0f03ce8f3: Pull complete
e319ab3d00f8: Pull complete
c097a9c08664: Pull complete
596df2728f7e: Pull complete
7d504dd10ea2: Pull complete
b1008cfb12ca: Pull complete
9d3ca1902f6d: Pull complete
882bbd427e4d: Pull complete
Digest: sha256:023a97dee605ba41dc61ce132e7814ebcd64843aeb01e844528467ed84802faa
Status: Downloaded newer image for ghcr.io/immich-app/immich-cli:latest
Checking connectivity with Immich instance...
Checking credentials...
Login status: OK
Successful authentication for user undefined
Indexing local assets...
/usr/src/app/bin/index.js:154
            if (supportedFiles['video'].includes(fileExtension) || supportedFiles['image'].includes(fileExtension)) {
                                        ^

TypeError: Cannot read properties of undefined (reading 'includes')
    at /usr/src/app/bin/index.js:154:41
    at Generator.next (<anonymous>)
    at fulfilled (/usr/src/app/bin/index.js:29:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Reply by alextran1502** [reply]
@narenarora that is the old cli, please use the new cli instead. You can find the instruction in the documentation

**Reply by narenarora** [reply]
@alextran1502 Thank you! Just checked that a new CLI app is available now. Do you know if there is a docker image for the new CLI? I didn't see any mention of it in the docs in a quick look. I still have to take a deeper look at the docs though.
